doc/flatpak-override.xml

```diff
@@ -326,6 +326,16 @@ key=v1;v2;
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--command=COMMAND</option></term>
+
+                <listitem><para>
+                    Execute the string <arg choice="plain">COMMAND</arg> to launch the application,
+                    which must be an executable name followed by zero or more arguments.
+                    This override takes maximum precedence, but existing launch arguments are preserved.
```

Comment:
As noted above, I think this is the wrong precedence: `flatpak run --filesystem=/foo` has higher precedence than `flatpak override --nofilesystem=/foo`, and I think the equivalent should be true for this new feature.